ndnsd/discovery/service-discovery.hpp

```diff
@@ -248,6 +255,7 @@ class ServiceDiscovery
   uint8_t m_counter;
 
   uint32_t m_syncProtocol;
+  uint32_t m_continuousDiscovery;
```

Comment:
What does this mean?

Reply:
Its a flag specific to a consumer application, if set, will not kill consumer application after fetching service info and so consumer will keep listening for service updates.